src/components/app/footer.tsx

```diff
@@ -137,7 +141,7 @@ const FooterNavigation: FunctionComponent = () => {
   )
 }
 
-const Footer: FunctionComponent = () => {
+const Footer: FunctionComponent<React.PropsWithChildren<unknown>> = () => {
```

Comment:
this is all codemod generated which results in a crufty blah, but fixes the breaking issue.